src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -24,7 +29,12 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("OrderServiceImpl getAllProductsFromShoppingCart method is done "
+                + "for userId: {}, productList is: {}",
+                userId, products.stream()
```

Comment:
Why do you need so complex computation just to write message to log? A lot of such logs can decrease your performance badly.